Source/GoogleMapsCarbonFootprint.js

```diff
@@ -77,18 +73,17 @@ function insertFootprint(route) {
  *   - trees: amount of trees to offset the CO2 emission in one year of growth
  */
 function createElement(footprint, trees) {
-  var e = document.createElement("div");
+  var e = document.createElement('div');
   var treesStr = treesToString(trees);
-  e.innerHTML = " <a href='http://goo.gl/yxdIs' target='_blank' title='"+ 
-                treesStr + 
-                "' class='carbon' id='carbon'>" + 
+  e.innerHTML = ' <a href=\'http://goo.gl/yxdIs\' target=\'_blank\' title=\'' +
+                treesStr +
+                '\' class=\'carbon\' id=\'carbon\'>' +
                 footprintToString(footprint) +
-                "</a> <a class='offset-link' href='http://goo.gl/yxdIs' target='_blank' title='"+ 
-                treesStr + "'>offset</a>"  
+                '</a> <a class=\'offset-link\' href=\'http://goo.gl/yxdIs\' target=\'_blank\' title=\'' +
```

Comment:
Thanks. There it says that ' should be preferred over ", but it doesn't say that \' should be preferred over " for quotations inside strings. One can certainly argue that the preference of ' over " entails the preference of \' over ". But I was wondering if this was explicitly said in the style guide that you are using.

Thanks again!